tests/Diagnostics.HealthChecks.UnitTests/Extensions/TaskExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// Imported from https://github.com/dotnet/aspnetcore/blob/main/src/Shared/TaskExtensions.cs
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks.UnitTests.Extensions
+{
+	public static class TaskExtensions
+	{
+		public static string CreateMessage(TimeSpan timeout, string filePath, int lineNumber)
```

Comment:
Should this be private?

```suggestion
		private static string CreateMessage(TimeSpan timeout, string filePath, int lineNumber)
```

Reply:
Done! Thanks :)